web/packages/core/src/load-options.ts

```diff
@@ -117,6 +117,47 @@ export const enum WindowMode {
     Gpu = "gpu",
 }
 
+/**
+ * The handling mode of navigate_to_url website calls.
```

Comment:
`navigate_to_url` is an internal ruffle name and we should not expect people to know that. I'd perhaps replace this and the other references with something more straight to the point, like "The handling of links to open a browser window", "allows opening a browser window to links", "prompts for confirmation before opening a browser window for links", etc

Reply:
Yeah, that's right.
The Ruffle name is called like that because the ActionScript method is called navigateToURL as well (and getURL in AVM1). Therefore, another possible comment would be "The handling mode of navigateToURL & getURL website calls.".
Would you prefer that, or a more general description like in the examples you gave?

Reply:
Most of the people using Ruffle on their sites haven't developed in ActionScript in many years (if ever), so I agree that using general/plain English instead of referencing specific ActionScript functions is better. That said, it's not something I would block merging over 😛